arbitrum-docs/node-running/how-tos/data-availability-committee/partials/_2.2-das-configuration-options-storage.mdx

```diff
@@ -0,0 +1,10 @@
+A DAS can be configured to use one or more of four storage backends:
+
+- [AWS S3](https://aws.amazon.com/s3/) bucket
+- Files on local disk
+- [Badger](https://dgraph.io/docs/badger/) database on local disk
+- [IPFS](https://ipfs.tech/)
+
+If more than one option is selected, store requests must succeed to all of them for it to be considered successful, while retrieve requests only require one of them to succeed.
+
+If there are other storage backends you'd like us to support you can send us a message on [Discord](https://discord.gg/arbitrum), or use the “Request an update” button on top of this page.
```

Comment:
It's also (in theory) fairly easy for people to add support for backends they are interested in and contribute it back to the Nitro repo. We have one third party contribution in the works currently for this, see the below issue and PR:

* https://github.com/OffchainLabs/nitro/issues/1989#issuecomment-1832540609
* https://github.com/OffchainLabs/nitro/pull/1992

Reply:
Thanks 💪 . I've added that as part of the paragraph, instead of just requesting an update. [Here](https://nitro-docs-git-update-dac-documentation-offchain-labs.vercel.app/node-running/how-tos/data-availability-committee/deploy-a-das#storage-options).